src/main/java/net/bramp/ffmpeg/probe/FFmpegProbeResult.java

```diff
@@ -30,4 +32,9 @@ public FFmpegFormat getFormat() {
   public List<FFmpegStream> getStreams() {
     return ImmutableList.copyOf(streams);
   }
+
+  public List<FFmpegChapter> getChapters() {
+    if (chapters == null) return Collections.emptyList();
```

Comment:
Oh good catch this can be null. Could you also fix getStreams?

Reply:
streams fixed, however, not sure if there could be media file w/o streams